.teamcity/settings.kts

```diff
@@ -0,0 +1,76 @@
+import jetbrains.buildServer.configs.kotlin.v2019_2.*
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildSteps.script
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildSteps.powerShell
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildFeatures.pullRequests
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildFeatures.xmlReport
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildFeatures.XmlReport
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildSteps.nuGetPublish
+import jetbrains.buildServer.configs.kotlin.v2019_2.triggers.vcs
+import jetbrains.buildServer.configs.kotlin.v2019_2.vcs.GitVcsRoot
+
+version = "2021.2"
+
+project {
+    buildType(Shimgen)
+}
+
+object Shimgen : BuildType({
+    id = AbsoluteId("Shimgen")
+    name = "Build"
+
+    artifactRules = """
+    """.trimIndent()
+
+    params {
+        password("env.EAZFUSCATOR_NET_LICENSE", "credentialsJSON:cdac4820-7cd7-54e4-b539-8ecaa3490a3c", display = ParameterDisplay.HIDDEN, readOnly = true)
+        param("env.vcsroot.branch", "%vcsroot.branch%")
+        param("env.Git_Branch", "%teamcity.build.vcs.branch.Shimgen_ShimgenVcsRoot%")
+        param("teamcity.git.fetchAllHeads", "true")
+    }
+
+    vcs {
+        root(DslContext.settingsRoot)
+
+        branchFilter = """
+            +:*
+        """.trimIndent()
+    }
+
+    steps {
+        powerShell {
+            name = "Prerequisites"
+            scriptMode = script {
+                content = """
+                    choco install windows-sdk-7.1 netfx-4.8-devpack visualstudio2022-workload-manageddesktopbuildtools --confirm --no-progress
```

Comment:
Correct me if I'm wrong, but we don't need the SDK or the devpack here do we?

Reply:
What makes you say this? While this "might" already be on the build agent, the suggestion from @JPRuskin was to include everything that is required by the build, and this will just no-op when run.

Reply:
For some reason I was thinking the devpack was installed by something else, but that may not actually be the case.

I have however just built this on my Dev VM without `windows-sdk-7.1` installed, so that one can probably go?

Reply:
We also have these packages installed for the Chocolatey/choco repository builds. I'm going to resolve this conversation as the build will work with it, and we can address cleanup and/or standardization of build configurations at a later date.